cadctl/cmd/investigate/investigate.go

```diff
@@ -69,6 +68,7 @@ func run(cmd *cobra.Command, args []string) error {
 	alertType := isAlertSupported(pdClient.GetTitle(), pdClient.GetServiceName())
 	if alertType == "" {
 		fmt.Printf("Alert is not supported by CAD: %s", pdClient.GetTitle())
+		pdClient.EscalateAlert()
```

Comment:
return value not checked (see pipeline)

Reply:
Indeed, thanks for pointing it out. Fixed this and the above. Unfortunately without https://issues.redhat.com/browse/OSD-15137 I can't run the lint job locally unless I figure out how to run it in a container with the correct go version, because the golangci-lint version is incompatible with my go 1.19.